fooof/synth.py

```diff
@@ -17,6 +17,96 @@
 
 SynParams = namedtuple('SynParams', ['background_params', 'gaussian_params', 'nlv'])
 
+class Stepper():
```

Comment:
Add a class methods to check Stepper values:

Note: use a static method, attached to the Stepper object.
```
@staticmethod
def _check_values(start, stop, step):
    if not start < stop:
        raise ValueError()
    if not step < (stop - start):
        raise ValueError()
```